application/src/main/java/org/togetherjava/tjbot/features/moderation/TransferQuestionCommand.java

```diff
@@ -123,14 +122,14 @@ public void onModalSubmitted(ModalInteractionEvent event, List<String> args) {
     }
 
     private static String createTitle(String message) {
-        if (message.length() >= TITLE_GUESS_MAX_LENGTH) {
-            int lastWordEnd = message.lastIndexOf(' ', TITLE_GUESS_MAX_LENGTH);
+        if (message.length() >= TITLE_MAX_LENGTH) {
+            int lastWordEnd = message.lastIndexOf(' ', TITLE_MAX_LENGTH);
 
             if (lastWordEnd == -1) {
-                lastWordEnd = TITLE_GUESS_MAX_LENGTH;
+                lastWordEnd = TITLE_MAX_LENGTH;
             }
 
-            message = message.substring(0, lastWordEnd);
+            message = message.substring(0, lastWordEnd).replace("\n", " ");
```

Comment:
```suggestion
            message = message.substring(0, lastWordEnd).replace('\n', ' ');
```
when only replacing char by char, prefer the overload that takes char instead of string. so `replace('\n', ' ')`. mini performance boost and just a tad cleaner